spec/honeycomb/integrations/rack_spec.rb

```diff
@@ -1,31 +1,41 @@
 # frozen_string_literal: true
 
 if defined?(Honeycomb::Rack)
+  # Rack::Session was moved to a separate gem in Rack v3.0
+  if Gem::Version.new(::Rack.release) >= Gem::Version.new("3.0")
+    require "rack/session"
+  end
   require "rack/test"
-  require "rack/lobster"
   require "warden"
 
+  class AnApp
+    def call(_env)
+      [200, { "content-type" => "text/plain" }, ["Hello world!"]]
+    end
+  end
```

Comment:
Huge fan of [replacing Rack::Lobster](https://github.com/honeycombio/beeline-ruby/pull/208/files#diff-9a936bf6526ca7d7934829e5fe3999ba63ad4e40a1160c586586b65d7d6d8401R15-R19)—as cute at it is—with a simple test app of our own to reduce dependencies.